uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Upgrader/Step/Step1104.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+class CRM_HRCore_Upgrader_Step_Step1104 {
```

Comment:
@davialexandre do you think it would be good to have an `UpgraderStepInterface` just to help the IDE with auto-completion?

Reply:
yup